README.md

```diff
@@ -1,26 +1,63 @@
-# Nimiq Go Client
+# Nimiq Go RPC Client
+[![Report card](https://goreportcard.com/badge/github.com/nimiq-community/go-client)](https://goreportcard.com/report/github.com/nimiq-community/go-client)
+[![GoDoc](https://godoc.org/github.com/nimiq-community/go-client?status.svg)](https://godoc.org/github.com/nimiq-community/go-client)
+
 
 > Go implementation of the Nimiq RPC client specs.
 
-## Usage
+## About
+A Nimiq RPC client library in Go. This client library implements the [Nimiq RPC specification](https://github.com/nimiq/core-js/wiki/JSON-RPC-API#remotejs-client). This client uses the jsonrpc library to handle JSON-RPC 2.0 requests and responses. For more information about this library see the [jsonrpc documentation](https://godoc.org/github.com/ybbus/jsonrpc)
+
+### What is Nimiq?
```

Comment:
I think we can remove this, people using this library should already know :-)

Reply:
True. Added it for completeness, but is probably redundant information for someone who wants to use the library.